src/main/java/dev/emi/trinkets/api/LivingEntityTrinketComponent.java

```diff
@@ -98,12 +98,12 @@ public boolean isEquipped(Predicate<ItemStack> predicate) {
 
 	@Override
 	public List<Pair<SlotReference, ItemStack>> getEquipped(Predicate<ItemStack> predicate) {
-		List<Pair<SlotReference, ItemStack>> list = new ArrayList<Pair<SlotReference, ItemStack>>();
```

Comment:
What's the purpose of erasing generics here?

Reply:
It's a well known feature in java that in an assignment like these you can leave them out. Having it twice is redundant and doesn't read any better imo. Tbh I was just fixing IDE warnings here.

Reply:
Sounds good to me, since you changed it in the whole codebase I don't have to worry about inconsistency and will just switch to writing like this, it saves me time